src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_DESCRIPTION = new Prefix("d/");
+    public static final Prefix PREFIX_DEADLINE = new Prefix("D/");
```

Comment:
maybe can change deadline prefix to "dl/" so that we standardise lower case 
```suggestion
    public static final Prefix PREFIX_DEADLINE = new Prefix("dl/");
```

Reply:
Point taken I have changed all token to lower case